src/System.Management.Automation/engine/PropertyAccessor.cs

```diff
@@ -318,6 +324,16 @@ internal override void SetDefaultSourcePath(string defaultPath)
             WriteValueToFile<string>(fileName, "DefaultSourcePath", defaultPath);
         }
 
+        /// <summary>
+        /// Allow loading assemblies by searching in GAC.
+        /// </summary>
+        /// <returns>Boolean indicating whether assemblies should be loaded from the GAC</returns>
+        internal override bool GetAllowGacLoading()
+        {
+            string fileName = Path.Combine(psHomeConfigDirectory, configFileName);
+            return ReadValueFromFile<bool>(fileName, "AllowGacLoading");
```

Comment:
@joeyaiello did we agree to make this opt-out vs opt-in?  I think that for most customers, they won't know to add a setting to a config file and will give up on PSCore6.  Perhaps we should default to opt-out and document this is best effort?